.buildkite/pipeline.yml

```diff
@@ -79,7 +79,7 @@ steps:
               context: "UI Tests (iPhone)"
 
       - label: "🔬 :jetpack: UI Tests (iPad)"
-        command: .buildkite/commands/run-ui-tests.sh 'iPad Pro 13-inch (M4)'
+        command: .buildkite/commands/run-ui-tests.sh 'iPad (10th generation) (17.5)'
```

Comment:
We are going to temporarily test on 17.5 unless until the `.sidebar` FF is removed and the tests are updated, which is something I'm working on right now.